extensions/panache/rest-data-panache/runtime/src/main/java/io/quarkus/rest/data/panache/runtime/jta/TransactionalExecutor.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.rest.data.panache.runtime.jta;
+
+import java.util.function.Supplier;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.transaction.Transactional;
+
+@ApplicationScoped
```

Comment:
Can this be made `@Singleton`?
We generally prefer it as it has less overhead. If it can't, no worries

Reply:
It can. I'll change it.